source/configuring_domain/01_managing_security/saml.rst

```diff
@@ -48,12 +48,13 @@ There are 2 methods for linking an OpenDataSoft user account:
 .. image:: images/saml__validation-link--en.png
     :alt: Account registration in SAML enabled domains
 
-Automatic Single Sign On
-------------------------
+Default login page selection
+----------------------------
 
-The OpenDataSoft platform offers a mechanism to automatically log in users when they first visit the domain.
+The OpenDataSoft platform allows to choose the login page that will be displayed to users when they click on a login link
+or try to access a restricted page.
 
-It works by starting the SAML authentication flow for anonymous users, as if they had clicked "log in with SAML", as soon as they encounter (almost) any page in the domain. This feature only makes sense for private domains as it prevents anonymous access to the domain. When Automatic Single Sign On is activated, users who wish to connect to the platform using their OpenDataSoft credentials have the option to do so by manually visiting the domain login page at ``https://<platform-url>/login/``
+If the IdP login page is selected as default, the SAML authentication flow will be initiated automatically if an anonymous user clicks on the login link or try to access a restricted page like the backoffice. When the IdP login page is selected, users who wish to connect to the platform using their OpenDataSoft credentials have the option to do so by manually visiting the domain login page at ``https://<platform-url>/login/``.
```

Comment:
"If the IdP login page is selected as default, the SAML authentication flow will be initiated automatically if an anonymous user clicks on the login link or **tries** to access a restricted page like the **back office**."